spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowControllerAutoConfiguration.java

```diff
@@ -124,6 +125,7 @@ public MavenResourceLoader MavenResourceLoader(MavenProperties properties) {
 	}
 
 	@Bean
+	@ConditionalOnMissingBean
```

Comment:
...although strictly speaking, the deployment controller should not care if it's a `DelegatingResourceLoader`, and in a case like k8s, it may only need `DockerResourceLoader` anyways. But the only other way to have conditionality (since it won't be the only `ResourceLoader